script.skin.helper.service/resources/lib/PluginContent.py

```diff
@@ -1111,7 +1158,7 @@ def getCast(movie=None,tvshow=None,movieset=None,episode=None,downloadThumbs=Fal
             cachedataStr = xbmc.getInfoLabel("ListItem.Title")+xbmc.getInfoLabel("ListItem.FileNameAndPath")+str(downloadThumbs)
     except: pass
     
-    cachedata = WINDOW.getProperty(cachedataStr).decode("utf-8")
+    cachedata = WINDOW.getProperty(cachedataStr+"bla").decode("utf-8")
```

Comment:
oops, and I was wondering why the cache was giving me a hard time. will correct it right away.